app/javascript/components/mentoring/queue/Solution.jsx

```diff
@@ -14,46 +13,28 @@ export function Solution({
   status,
   updatedAt,
   url,
+  onMouseEnter,
+  onMouseLeave,
 }) {
-  const [referenceElement, setReferenceElement] = useState(null)
-  const [popperElement, setPopperElement] = useState(null)
-  const [showPopper, setShowPopper] = useState(false)
-  const { styles, attributes } = usePopper(referenceElement, popperElement)
-
   return (
-    <>
-      <tr
-        ref={setReferenceElement}
-        onMouseEnter={() => setShowPopper(true)}
-        onMouseLeave={() => setShowPopper(false)}
-      >
-        <td>
-          <TrackIcon track={{ title: trackTitle, iconUrl: trackIconUrl }} />
-        </td>
-        <td>
-          <img
-            style={{ width: 100 }}
-            src={menteeAvatarUrl}
-            alt={`avatar for ${menteeHandle}`}
-          />
-        </td>
-        <td>{menteeHandle}</td>
-        <td>{exerciseTitle}</td>
-        <td>{isStarred.toString()}</td>
-        <td>{haveMentoredPreviously.toString()}</td>
-        <td>{status}</td>
-        <td>{fromNow(updatedAt)}</td>
-        <td>{url}</td>
-      </tr>
-      {showPopper && (
-        <div
-          ref={setPopperElement}
-          style={styles.popper}
-          {...attributes.popper}
-        >
-          Lorem ipsum
-        </div>
-      )}
-    </>
+    <tr onMouseEnter={onMouseEnter} onMouseLeave={onMouseLeave}>
```

Comment:
We need to expose the HTML reference used in this component as that is required for the propper popper (pun intended) positioning in the parent component.